templates/newrelic-stack-set.yml

```diff
@@ -3,8 +3,9 @@ Description: Integrates New Relic with AWS accounts that use AWS Control Tower.
 Parameters:
   NewRelicAccountNumber:
     Type: String
-    Description: New Relic account number.
+    Description: Your New Relic account ID
```

Comment:
Please don't use "Your". It is self-explanatory that user needs to enter their account id and not someone elses ;)